editor/src/clj/editor/scene_picking.clj

```diff
@@ -41,3 +44,21 @@
                       (coll/some #(and (not= % (:node-id node))
                                        (contains? node-ids %)))))]
     (e/remove child? nodes)))
+
+(defn contains-resource?
```

Comment:
This doesn't really belong in `scene_picking.clj`. "Picking" in this case refers to details related to identifying a clicked object in the Scene View.

Maybe this function would belong better in `defold_project.clj`? We probably want to rename it to something like `node-refers-to-resource?` if we do, since `project/contains-resource?` suggests it does something else. 😅